pom.xml

```diff
@@ -75,6 +76,7 @@
     <sonar.organization>acrolinx</sonar.organization>
     <sonar.projectKey>acrolinx_flexmark-java</sonar.projectKey>
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.coverage.jacoco.xmlReportPaths>../flexmark-coverage/target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
sure that this `../` path here is correct? If we are already in the root, why do we need to go to the parent directory first?

Reply:
When executing this the coverage is missing since it runs for all the submodules
<img width="1212" alt="Screenshot 2024-10-30 at 10 30 19" src="https://github.com/user-attachments/assets/5ae3f424-0783-41a5-9830-5417bbea7007">